### PR TITLE
Add events

### PR DESCRIPTION
**Pull request by LYANGEND**
**In this project, I achieved the following milestones**

- Copy the files calculate.js and operate.js into a logic/ directory in your project.

- Analyze the files calculate.js and operate.js,  add big.js(https://www.npmjs.com/package/big-js) as a dependency).

- Import the files in your Calculator component.

- Implement the event handlers you need to use the math logic from calculate.js and operate.js in your React component.

-  use state and props, pass props to child components, use lifecycle methods, lift state up.

- Test the app in the browser. Performs all math operations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ To learn React, check out the [React documentation](https://reactjs.org/).
```
Comment (arslanbisharat) [inline README.md]: - [ ] Add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (peterrobert) [reply README.md]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @LYANGEND, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**